apps/frontline-widgets/src/app/messenger/components/faq/components/CategoryView.tsx

```
@@ -0,0 +1,157 @@
+              onClick={() => handleSubCategoryClick(subCat._id)}
+              role="button"
+              tabIndex={0}
+              onKeyDown={(e) =>
+                e.key === 'Enter' && handleSubCategoryClick(subCat._id)
+              }
+            >
+              <span
+                className="shrink-0 p-2 rounded-xl flex items-center justify-center"
+                style={{
+                  backgroundColor:
+                    `color-mix(in srgb, ${details?.color} 60%, transparent)` ||
```

ESLint

[error] Unexpected constant truthiness on the left-hand side of a `||` expression.

(no-constant-binary-expression)